test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -216,6 +216,8 @@ public static ObservableTool invokeAsync(Map<String, String> env, InputStream st
         ProcessBuilder pb = new ProcessBuilder(args);
         if (env != null && !env.isEmpty())
             pb.environment().putAll(env);
+        String jvmOpts = pb.environment().getOrDefault("JVM_OPTS", "") + " -Dcassandra.disable_tcactive_openssl=true";
```

Comment:
why not put `-Dcassandra.disable_tcactive_openssl=true` into the build.xml instead?
 e.g. like here https://github.com/apache/cassandra/blob/trunk/build.xml#L1467-L1495

Reply:
well, because we want the subprocess also use that property - it is not inherited automatically, right?